_content/applications/tago/index.md

```diff
@@ -44,7 +55,7 @@ To send data back to your device, just go to your device at Tago and visit the t
 
 Here you can add two parameters, for different data formats.
 
-**payload_raw** 
+**payload** 
```

Comment:
Sure?

Reply:
Yes, is different from TTN docs, but we use the same downlink format for all providers. So it must be only payload for raw downlinks.